Sources/FoundationExtensions/Promise/PromiseType+Zip.swift

```diff
@@ -372,4 +372,27 @@ extension PromiseType {
         }
     }
 }
+
+@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+extension Publisher {
+    public static func zip<P: PromiseType>(_ promises: [P]) -> Publishers.Promise<[P.Output], P.Failure>
+    where P.Output == Output, P.Failure == Failure {
+        switch promises.count {
+        case ...0:
```

Comment:
Just out of curiosity: Is it possible that count on an array returns a value less than 0?

Reply:
For sure it's not. I guess the previous implementation of this switch was catching `...0`, `1` and `2...` instead of `default`, in that case ...0 was needed to pattern match all possibilities of Int. Now it shouldn't matter anymore.